fab/inventory/enikshay

```diff
@@ -12,13 +12,13 @@
 [formplayer0] # formplayer, riak
 172.25.2.7
 
-[formplayer1] # formplayer
+[formplayer1] # formplayer, riak
```

Comment:
i think roughly splitting half for riak and half for formplayer like you did is likely fine. i just know nikshay heavily heavily uses formplayer. so far formplayer hasn't proven to be memory constrained